wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -382,6 +383,30 @@ public RepeatCommand repeatedly() {
     return new RepeatCommand(this);
   }
 
+  /**
+   * Decorates this command to run repeatedly, restarting until the command runs for the given times
+   * amount. The decorated command can still be canceled.
```

Comment:
```suggestion
   * Decorates this command to run the specified number of times, restarting it when it ends.
```
Cancellation is a property of all commands.

Reply:
see:
https://github.com/wpilibsuite/allwpilib/blob/c289562a06e23e612071bc5ff161ea68d7b912ca/wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java#L370-L371

I just coppied from the other `.repeatedly()` functions docs, should I change it for both or none?

Reply:
Hm, ok. I guess leave it in then